### ISSUE TITLE
test use any resolution frames

### ISSUE THREAD

**Issue by jian3xiao** [liuziwei7/voxel-flow#9]
Thank you for sharing your implementation. And I have a question. 
The model is trained on UCF101 frames resized to 256x256, but how I test the model using original size frames. The paper said , We train our method on the public UCF-101 dataset, but test it on a wide variety of videos. Our method can be applied at any resolution.
I guess the procedure like this when I want test 856x480 two frames. First frames are resized to 256x256 and feed into the model. Then the model output a voxel flow fields 256x256x3, and re-scaled it to flow_new=height(or width)*flow. Finally,  obtain the interpolationed frame by using bilinear_interp and mask.
I don't know if this process is correct, and how the real procedure when applied different size frames.

**Comment by liuziwei7** [comment]
Yes, the aforementioned procedure is correct. Since voxel flow is invariant to scaling, it can be readily applied to high-res videos.

**Comment by jian3xiao** [comment]
> Yes, the aforementioned procedure is correct. Since voxel flow is invariant to scaling, it can be readily applied to high-res videos.

Thanks a lot ! Which part of your code implement the procedure, could you help me point it out ? Is it the idea inspired from FCN? FCN can work on arbitrary sized images although the model is trained with resized image of 256x256.
